testing/test_rec.c

```diff
@@ -22,9 +22,9 @@ void test_sd_rec_create(void) {
     for(int i = 0; i < 10; i++) {
         sd_rec_move mv;
         fill((char *)&mv, sizeof(sd_rec_move));
-        mv.lookup_id = 2;
-        mv.action = SD_ACT_KICK | SD_ACT_PUNCH | SD_ACT_LEFT | SD_ACT_DOWN;
-        mv.extra_data = NULL;
+        smallbuffer_create(&mv.extra_data);
```

Comment:
isn't this smallbuffer_create unnecessary?

Reply:
Yes, because we eventually pass the `rec` into `sd_rec_free`, which calls smallbuffer_free.

`fill` is not `memset(0)`-- it fills the struct with ascending bytes 1, 2, 3..